src/main/java/core/basesyntax/service/filesoperation/ReadDataFromFileImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.filesoperation;
+
+import core.basesyntax.service.parsing.ParsingDataService;
+import core.basesyntax.service.parsing.ParsingDataServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadDataFromFileImpl implements ReadDataFromFile {
+    private static final String ERROR_READ_MESSAGE = "Can't find file by path: ";
+    private List<String> frits;
+
+    @Override
+    public void readData(String inputFile) {
+
+        ParsingDataService parsingDataService = new ParsingDataServiceImpl();
```

Comment:
Separate this logic.
Reader should only read from file, and return list with data